tests/controllers/test_registry_instrument.py

```diff
@@ -0,0 +1,196 @@
+from unittest import TestCase
+from unittest.mock import MagicMock, patch
+
+from application.controllers.registry_instrument import RegistryInstrument
+from application.exceptions import RasError
+
+
+class TestRegistryInstrumentController(TestCase):
+
+    @patch("application.controllers.registry_instrument.query_registry_instruments_by_exercise_id")
+    def test_get_registry_instruments_by_exercise_id_returns_list_of_dicts(self, mock_query):
```

Comment:
mock_query is a touch lazy, nice to have a true reflection of what it is. It's not so bad here, but when you have multiple patches it's difficult to keep track

Reply:
I generally name the mock after the patch, so if Im 10 lines down I know what it is rather than mock_query. So I do something like query_registry_instruments_by_exercise_id

Reply:
Oh sorry.. I thought you were saying I shouldn't be mocking the DB call at all :o) It was just a comment about the name. Sure, I'll update.